index.d.ts

```diff
@@ -16,10 +16,11 @@ export function getWalletBalance(config: Config): Promise<any>
 
 export type WalletOption = "Nami" | "Gero";
 
+/** Corresponds to the network tag from
+ * https://cips.cardano.org/cips/cip19/#networktag */
 export type NetworkId
-    = 2 // Preview
-    | 1 // Pre-production
-    | 0 // Mainnet
+    = 0 // Testnet
+    | 1 // Mainnet
```

Comment:
Is this correct? I thought 0 was mainnet

Reply:
Pretty sure it's correct, since it corresponds to the network tag (confusing naming):
- https://github.com/Plutonomicon/cardano-transaction-lib/blob//src/Serialization/Address.purs#L325-L329
- https://cips.cardano.org/cips/cip19/#networktag

Reply:
Fair enough